src/library/Uno.Themes.WinUI.Markup/Theme/CalendarDatePicker.cs

```diff
@@ -0,0 +1,117 @@
+using System;
+using Windows.UI;
+using Microsoft.UI.Text;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup;
+using Uno.Extensions.Markup.Internals;
+
+using ResourceKeyDefinitionAttribute = Uno.Themes.WinUI.Markup.ResourceKeyDefinitionAttribute;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static partial class CalendarDatePicker
```

Comment:
Is there a reason to generate these all as partial classes? Is this so we can add manual code to suppliment the generated code?

Reply:
Well, it doesnt hurt to add `partial`. But, we do have use-case for this, with Button.xaml and FAB.xaml, where both contains styles target `Button`-type.